google-cloud-bigquery/lib/google/cloud/bigquery/service.rb

```diff
@@ -61,7 +61,7 @@ def service
             service.client_options.open_timeout_sec = timeout
             service.client_options.read_timeout_sec = timeout
             service.client_options.send_timeout_sec = timeout
-            service.request_options.retries = @retries || 3
+            service.request_options.retries = 0 # handle retries manually
```

Comment:
A pointer might be helpful:
`# handle retries via backoff in #execute`

Reply:
That's too long for the line, so I changed it to `# handle retries in #execute`.